demovieworderhooks/README.md

```diff
@@ -0,0 +1,72 @@
+Demo View Order Hooks
+=====================
+
+This module was created in order to demonstrate how to use the new hooks introduced with the new "View an Order" back-office page in PrestaShop 1.7.7.0 .
+
+It uses the following hooks:
+- displayBackOfficeOrderActions
+- displayAdminOrderTabContent
+- displayAdminOrderTabLink
+- displayAdminOrderMain
+- displayAdminOrderSide
+- displayAdminOrder
+- displayAdminOrderTop
+- actionGetAdminOrderButtons
```

Comment:
hook `actionGetAdminOrderButtons` is not present in the migrated controller so suggest removing it from the list

Reply:
o_O
This is a bug !
I'll check that